packages/wallet/src/cip30.ts

```diff
@@ -116,7 +118,7 @@ export const createWalletApi = (
       if (error instanceof ApiError) {
         throw error;
       }
-      throw new ApiError(APIErrorCode.InternalError, 'Nope');
+      throw new ApiError(APIErrorCode.InternalError, (error as Error)?.message || UNKNOWN_ERROR);
```

Comment:
Seems to be duplicated quite a few times
```suggestion
      throw new ApiError(APIErrorCode.InternalError, formatUnknownErrorMessage(error));
```

Reply:
Fixed in [65a3784](https://github.com/input-output-hk/cardano-js-sdk/pull/602/commits/65a3784f00c6574efed79cbdcc530954c50b77d1)